### PR TITLE
Explain and clarify keys

### PR DESCRIPTION
**Pull request by vrotmanh**

### CODE REVIEW

**Comment by MaxBlaushild** [comment]
Please don’t hate me, but maybe move this to Linnia-resources and link to that from here? This seems useful to have in most of our repos

File: README.md [hunk 1]
```diff
@@ -86,18 +84,54 @@ Congrats your Linnia App is running!!
...
+- YOUR_ETHEREUM_ADDRESS
+- DATA_HASH_YOU_WANT
+
+```
+[LINNIA_BOX_URL]/permissions?viewerEncyptionPublicKey=[YOUR_ENCRYPTION_PUBLIC_KEY]&viewerEthereumAddress=[YOUR_ETHEREUM_ADDRESS]&dataHash=[DATA_HASH_YOU_WANT]
+```
+
+
+
+## Keys
```
[the remark this answers is not in the document]
Comment (vrotmanh) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -86,18 +84,54 @@ Congrats your Linnia App is running!!
...
+
+To learn more about the Keys that Linnia uses go [HERE](https://github.com/ConsenSys/linnia-resources/blob/master/KEYS.md)
```
[the remark this answers is not in the document]
Comment (vrotmanh) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -86,18 +84,54 @@ Congrats your Linnia App is running!!
...
+
+
+
+## Test Users
+
+In order to test Linnia you can use the following keys. We already uploaded data of 40 different diabetic patients (This is not real data)
+
+
+
+### **Ethereum Address**es
```
[the remark this answers is not in the document]
Comment (vrotmanh) [reply README.md]: No, we say in the daily standup that we are not going to give the Ethereum private keys of the test users. If you want to test that you should use the Faucet

[the remark this answers is not in the document]
Comment (vrotmanh) [reply README.md]: We keep the test addresses only to search I think

[the remark this answers is not in the document]
Comment (vrotmanh) [reply README.md]: Maybe for the hackhatons top

[the remark this answers is not in the document]
Comment (godfreyhobbs) [reply README.md]: yes test address just so you have a dummy account to share permissions with.

**Pull request merged**